### PR TITLE
MM-44114 : Announcing of channel interface language dropdown correctly

### PR DESCRIPTION
**Pull request by babinderrathi**
#### Summary
Announcing of Settings > Display > Language > Change interface language field dropdown correctly 

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-44114
Fixes https://github.com/mattermost/mattermost-server/issues/21159

#### Related Pull Requests
none
#### Screenshots

|  before | 

https://user-images.githubusercontent.com/102028591/189839616-ae976aa3-87eb-4d5b-8750-166a67e1af0b.mp4

|  after  |

https://user-images.githubusercontent.com/102028591/189838983-b1e9fb1a-0917-45eb-bf0f-776a95b64a02.mp4

```release-note

Channel interface language dropdown in  Settings > Display > Language > Change interface language field not announcing that it was a dropdown by ScreenReader. Now by giving Aria-label to it makes it announcing properly by ScreenReader.

```

### CODE REVIEW

File: components/user_settings/display/manage_languages/manage_languages.tsx [hunk 1]
```diff
@@ -179,11 +181,13 @@ export default class ManageLanguage extends React.PureComponent<Props, State> {
                 zIndex: 9999,
             }),
         };
+        const aria = intl.formatMessage({id: 'user.settings.languages.dropdown', defaultMessage: 'drop down selector to change the interface language'});
```
Comment (M-ZubairAhmed) [inline components/user_settings/display/manage_languages/manage_languages.tsx:184]: a better variable name can be `interfaceLanguageLabelAria`

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: i18n/en.json [hunk 2]
```diff
@@ -5085,6 +5085,7 @@
   "user.settings.import_theme.submit": "Submit",
   "user.settings.import_theme.submitError": "Invalid format, please try copying and pasting in again.",
   "user.settings.languages.change": "Change interface language",
+  "user.settings.languages.dropdown": "drop down selector to change the interface language",
```
Comment (M-ZubairAhmed) [inline i18n/en.json:5088]: we also suffix the translation with "ariaLabel" to help translators know its an aria label.
eg
```
file_upload.menuAriaLabel
insights.filter.ariaLabel

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Review by devinbinnie — APPROVED** [review]
LGTM other than @M-ZubairAhmed's comments. Thanks @babinderrathi!

File: i18n/en.json (hunk 2 above)
Comment (hmhealey) [reply i18n/en.json:5088]: Also, this is a bit nitpicking, but "dropdown" should probably be a single word in this case

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by M-ZubairAhmed** [comment]
@babinderrathi can we get this to finish line. its almost done

**Comment by babinderrathi** [comment]
@M-ZubairAhmed  Can you please re-review and suggest me to change if any, because I updated all requested changes.

**Comment by hmhealey** [comment]
Dismissing myself for review since we'll have two dev reviews already between Devin and Zubair

**Review by cwarnermm — APPROVED** [review]
Thank you, @babinderrathi!

**Review by M-ZubairAhmed — APPROVED** [review]

**Comment by jgilliam17** [comment]
Hi @babinderrathi 
The language dropdown is announced as expected.
There is an issue with navigations through the settings modal.
Example: 
WIth screen reader enabled, open settings modal, notice that the focus is on the last item, "words that trigger mentions", and tab to Display, select it Display tab again the focus is on the last item "Language"  
Expected: first item to be selected on both tabs

**Comment by amyblais** [comment]
@hmhealey @M-ZubairAhmed Can you help manually cherry-pick this to the ``release-7.1`` branch? I was requested to backport this to the v7.1 ESR.

**Pull request merged**